sample-compose/app/src/androidTest/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreenTest.kt

```diff
@@ -66,7 +67,9 @@ class HomeScreenTest {
         assertEquals(expectedAppDestination, AppDestination.Second)
     }
 
-    private fun initComposable(testBody: ComposeContentTestRule.() -> Unit) {
+    private fun initComposable(
+        testBody: AndroidComposeTestRule<ActivityScenarioRule<MainActivity>, MainActivity>.() -> Unit
```

Comment:
I'm not sure I understand this change, or was it to match the other `HomeScreenTest`? 😅

Reply:
@ryan-conway I forgot to use the same logic for this test class. We initializing a ComposeTestRule with `val composeRule = createAndroidComposeRule<MainActivity>()`, so the `initComposable` should handle `AndroidComposeTestRule` instead.

The benefit of using `AndroidComposeTestRule` instead of `ComposeContentTestRule` is having `activity` reference to fetch resource content, e.g. `activity.getString(R.string.error_generic)`.